plenum/test/node_request/test_timestamp/test_timestamp_post_view_change.py

```diff
@@ -0,0 +1,73 @@
+from stp_core.loop.eventually import eventually
+
+from plenum.test.helper import send_reqs_to_nodes_and_verify_all_replies, \
+    sendRandomRequests, checkViewNoForNodes
+from plenum.test.node_request.test_timestamp.helper import make_clock_faulty, \
+    get_timestamp_suspicion_count
+from plenum.test.test_node import ensureElectionsDone, getNonPrimaryReplicas
+from plenum.test.view_change.helper import ensure_view_change
+
+Max3PCBatchSize = 4
+
+from plenum.test.batching_3pc.conftest import tconf
+
+# lot of requests will be sent and multiple view changes are done
+TestRunningTimeLimitSec = 200
+
+
+def test_new_primary_has_wrong_clock(tconf, looper, txnPoolNodeSet, client1,
+                                wallet1, client1Connected):
+    """
```

Comment:
We can have it, but its a better test since it shows the transition which is a more common occurrence, i can check for the just the clock being bad of a node when the network's life started but this kind of scenario is more common where a running node's clock goes bad